src/PhpcsDiff.php

```diff
@@ -55,18 +57,43 @@ public function __construct(array $argv, CLImate $climate)
             $this->isVerbose = true;
         }
 
-        if (!isset($this->argv[1])) {
-            $this->error('Please provide a <bold>base branch</bold> as the first argument.');
-            return;
+        $this->ruleset = $this->getOption('--ruleset', self::DEFAULT_RULESET);
+
+        $this->baseBranch = $this->getArgument(0, '');
+        $this->currentBranch = $this->getArgument(1, '');
+
+        if ($this->isVerbose) {
+            if (!$this->baseBranch && !$this->currentBranch) {
+                $this->climate->comment("Comparing unstaged changes.");
+            } else {
+                $this->climate->comment(
+                    'Comparing branch: "' . $this->baseBranch . '" against: "' . $this->currentBranch . '"'
+                );
+            }
         }
+    }
 
-        $this->baseBranch = 'origin/' . str_replace('origin/', '', $this->argv[1]);
-        $this->currentBranch = trim(shell_exec('git rev-parse --verify HEAD'));
+    protected function getArgument($index, $default = null)
+    {
+        $arguments = array_filter($this->argv, function ($val) {
+            return strpos($val, '-') === false;
```

Comment:
Could we change this to use yoda-style conditions?
```suggestion
            return false === strpos($val, '-');
```